### ISSUE TITLE
Shouldn't the TonelRepositoryTest be running the inherited tests as well as the tests defined for Tonel?

### ISSUE THREAD

**Issue by dalehenrich** [pharo-vcs/tonel#13]
[Only two tests are being run, but there are a number of other tests that could be inherited](https://travis-ci.org/pharo-vcs/tonel/jobs/299118429#L107-L109).

I think that `shouldInheritSelectors` needs to be implemented to return `true` on the class-side to run the tests ...

**Comment by dalehenrich** [comment]
Hmm, I was looking Pharo7.0, above, for Pharo6.0 and my most recent modifications, [there are more tests being run](https://travis-ci.org/GsDevKit/tonel/jobs/299399607#L834-L843) and [not all of the inherited tests are passin](https://travis-ci.org/GsDevKit/tonel/jobs/299399607#L1030-L1042) ... odd that different builds result in different tests being run ... ...

I'll add `shouldInheritSelectors`, because GemStone needs that anyway ...

**Comment by dalehenrich** [comment]
... on another note, each of the failing tests would need a port to Tonel I think, since they seem to be testing the Monticello meta data aspects --- so I've wired them out and flagged as a `#todo`

**Comment by estebanlm** [comment]
yeah, I didn't port them explicitly since they seemed to no have sense in the tonel context.
